docs/content/ecs/entity-contracts.md

```diff
@@ -23,7 +23,7 @@ A reactive component is implemented as a temporary component, and so has the sam
 
 ## Components on ECS Entities that represent SpatialOS entities
 
-For each SpatialOS entity that a [worker]({{urlRoot}}/content/workers/workers-in-the-gdk) checks out, the `SpatialOSReceiveSystem` creates an ECS Entity for that worker’s ECS world.
+For each SpatialOS entity that a [worker]({{urlRoot}}/content/workers/workers-in-the-gdk) checks out, the `EcsViewSystem` creates an ECS Entity for that worker’s ECS world.
```

Comment:
Do we need to say which system? That seems like an implementation detail until we offer users the ability to do it with arbitrary ones.